black_it/MCS.R

```diff
@@ -0,0 +1,105 @@
+# Reading the data
+
+mData <- read.csv("Data.csv")
+attach(mData)
+
+fnComplete <- function(x,l=301) {
+	return(c(x,rep(x[length(x)],l-length(x))))
+}
+
+# Building the MCS
+
+vZ <- 300+4.9*(1:301)
+
+mDist <- NULL
+for (i in 1:16) {
```

Comment:
What does this for loop do? and the following one? Just two likes of comments could be really helpful